paper.tex

```diff
@@ -244,6 +244,8 @@ \subsubsection{Components}
 \item \textbf{Ethereum Smart Contract: }
 Smart contract is a protocol for formulating agreement on a blockchain that provides verification and execution of the contract. The code in the smart contract can interact with other contracts, make decisions, store data and transfer cryptocurrency. All conditions and states established in the contract are transparent and with enforcement. The appearance of smart contracts makes trading more flexible, and achieves more complex trading patterns in reality.
 
+The Product Contract is used during the trading process, where Product Contract records all the details of data products, such as MAM channel root, blinded session key and data providers' DID Document address. Furthermore, data providers and consumers exchange session key through Product Contract with the help of brokers. Though this design may cost more transaction fee than doing so off-chain, it is considered a better strategy to assure the profit of both providers and consumers. The further details of key exchangement will be illustrated in Section~\ref{section:trading}.
```

Comment:
Shall we use "channel keys" rather than "channel root"? According to [MAM](https://github.com/iotaledger/MAM), the channel keys are the merkle root of the set of one-time keys used to sign the messages.

Reply:
I think changing "channel root" to "channel ID" will be more precise.